File: eAnimalShelter/UI/eanimalshelter_desktop/lib/models/requests/announcement_update_request.dart
class AnnouncementUpdateRequest {
  String title;
  String content;
  String? imageUrl;
  bool isActive;

  AnnouncementUpdateRequest({
    required this.title,
    required this.content,
    this.imageUrl,
    required this.isActive,
  });

  Map<String, dynamic> toJson() {
    return {
      "title": title,
      "content": content,
      "imageUrl": imageUrl,
      "isActive": isActive,
    };
  }
}

File: eAnimalShelter/eAnimalShelter.Model/Requests/AnnouncementUpdateRequest.cs
namespace eAnimalShelter.Model.Requests
{
    public class AnnouncementUpdateRequest
    {
        public string Title { get; set; } = string.Empty;

        public string Content { get; set; } = string.Empty;

        public string? ImageUrl { get; set; }

        public bool IsActive { get; set; }
    }
}
